src/base/utils/misc.cpp

```diff
@@ -237,52 +237,52 @@ QPoint Utils::Misc::screenCenter(const QWidget *w)
 }
 #endif
 
-QString Utils::Misc::unitString(Utils::Misc::SizeUnit unit)
+QString Utils::Misc::unitString(Utils::Misc::SizeUnit unit, bool isSpeed)
```

Comment:
I would add `const` to these parameters.

Reply:
done

Reply:
@Chocobo1, when you suggest to add const to parameter passed by value, please, don't forget to mention that it's applicable only to the function definition (and shouldn't affect its declaration).